app/src/main/java/ar/scacchipa/twittercloneapp/data/UserAccessTokenMapper.kt

```diff
@@ -0,0 +1,20 @@
+package ar.scacchipa.twittercloneapp.data
+
+import ar.scacchipa.twittercloneapp.utils.Constants
+
+class UserAccessTokenMapper: IMapper<UserAccessTokenData, Credential> {
```

Comment:
This mapper does not make sense to me. Both types are types from the data layer. It is not the idea of a mapper. The mapper should transmform data types between layers.

Reply:
I think your last coment is old. I think you got it right now, Credential is from domain layer and UserAccesToken from data, which is ok.